WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -874,10 +884,19 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
                 blogPickerWasPressed()
             case .autoSave:
                 break
+                // Inelegant :(
+            case .continueFromHomepageEditing:
+                continueFromHomepageEditing()
+                break
             }
         }
     }
 
+    // Not ideal, but seems the least bad of the alternatives
+    @objc func continueFromHomepageEditing() {
+        fatalError("This method must be overriden by the extending class")
+    }
```

Comment:
From the current patterns in place for code reuse, this was the best way I could manage reusing the abstraction here without resorting to much larger refactoring. Welcome alternatives if there is a better way to achieve a similar result though.